docs/RELEASE.md

```diff
@@ -0,0 +1,54 @@
+# Release process and check offs (client only)
+______
+since there is no server component at this time
+
+## Prepare release
+1. [ ]  User testing on mockups (design)
+1. [ ]  go/nogo on individual features (engineering)
+1. [ ]  create release tag for the selected features
```

Comment:
Only tag, no branch?  Should we branch so ppl can make any required security/legal fixes on an older release?

Reply:
that's a good question. Do we need to keep supporting older releases? this is for a phone app, so once we release the next version (say, version 1), then the autoupdate process will update all phones. Why would we want to make fixes on an older release that nobody is using?

Reply:
Filed #279